src/lib/wordpress-provider/playground-cli/playground-server-process-child.ts

```diff
@@ -157,8 +157,22 @@ async function startServer(
 			args.wp = serverOptions.wordPressVersion;
 		}
 
+		const defaultConstants = {
+			WP_SQLITE_AST_DRIVER: true,
+		};
+
 		if ( options.blueprint ) {
-			args.blueprint = options.blueprint;
+			args.blueprint = {
```

Comment:
Playground CLI doesn't provide any interface to set constants, so we need to alter the blueprint for that purpose.